### DISCUSSION TITLE
Can I split my schema.prisma one file into several files like schema1.prisma, schema2.prisma, schema3.prisma

### DISCUSSION THREAD

**Discussion by 45592858** [prisma/prisma#4289] [Q&A] [answered]
As one schema.prisma file, it is so bigger ...

**Comment by ryands17** [comment, marked as answer]
@45592858 👋 
Unfortunately this is not possible, but there's a feature request [here](https://github.com/prisma/prisma/issues/2377) for the same.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
